object_detect/PotInfo[] potInfos
